src/index.html

```diff
@@ -6,13 +6,73 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <title>Product cards</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="body">
+    <div
+      class="product-card"
+      data-qa="card"
+    >
+      <img
+        class="product-card__image"
+        src="./images/imac.jpeg"
+        alt="imac"
+      />
+
+      <div class="product-card__content">
+        <h2 class="product-card__title">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </h2>
+      </div>
+
+      <div class="product-card__code">
+        <p class="product-card__code_text">Product code: 195434</p>
+      </div>
+
+      <div class="product-card__reviews">
+        <div class="product-card__stars">
+          <div class="product-card__star"></div>
```

Comment:
Add the modifier class and fix classes
```suggestion
        <div class="stars stars--4">
          <div class="stars__star"></div>
```